Stylesheets/Docs/avatar/docs_avatar.html

```diff
@@ -7,61 +7,76 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Avatar | MondayBlues</title>
     <link rel="stylesheet" href="/Stylesheets/Abstract/fontawesome-free-6.0.0-web/css/all.css">
+    <link rel="shortcut icon" href="Assets/images/logo-modified.png" type="image/x-icon">
     <link rel="stylesheet" href="/Stylesheets/Docs/docs.css">
 </head>
 
 <body>
     <div class="page-contents">
         <nav class="display-align-center display-justify-space-between">
-            <div class="heading display-align-center display-justify-space-between">
+            <div class="heading display-align-center">
                 <div class="hamburger-nav clickable-image">
                     <div class="bars"></div>
                     <div class="bars"></div>
                     <div class="bars"></div>
```

Comment:
use span instead of div as `span` is inline element - easy to style.

Reply:
Thankyou for pointing this out